request/file_list.proto

```diff
@@ -26,9 +26,11 @@ message FileListResponse {
     string directory = 3;
     // Directory parent (null/empty if top-level)
     string parent = 4;
-    // List of available image files, with file type information and size information.
+    // If FileListFilterMode is NOT "AllFiles": List of available image files with file type, size, and modified date information.  
+    // If FileListFilterMode is "AllFiles": List of available image files with size and modified date information. Type information are sent when requested.  
```

Comment:
`Type information are sent when requested.` -- `are` should be `is`, but also this is confusing (requested when?). I suggest rephrasing like this (which also removes the repetition):

*List of available image files with sizes and modification dates. If FileListFilterMode is not "AllFiles", file types are also included.*